### PR TITLE
Added Product and Division methods

### PR DESCRIPTION
**Pull request by Pavithran2612**

### CODE REVIEW

**Review by Salman-003 — CHANGES_REQUESTED** [review]
make proper alignment

File: Calculator/src/CalculatorMain.java [hunk 1]
```diff
@@ -11,14 +11,15 @@ public static void main(String args[])
 		CalculatorMain obj = new CalculatorMain();
 		obj.display();
 		Scanner sc = new Scanner(System.in);
```
Comment (Pavithran2612) [inline Calculator/src/CalculatorMain.java:13]: ok

File: Calculator/src/CalculatorMain.java [hunk 2]
```diff
@@ -11,14 +11,15 @@ public static void main(String args[])
...
-		System.out.println("Please choose the operation :");
+		System.out.print("Please choose the operation to perform: ");
 		int choice = sc.nextInt();
-		System.out.println("Please enter the value 1");
+		System.out.print("\nPlease enter the value 1: ");
 		double num1 = sc.nextInt();
-		System.out.println("Please enter the value 2");
+		System.out.print("\nPlease enter the value 2: ");
```
Comment (Salman-003) [inline Calculator/src/CalculatorMain.java:18]: make proper alignment

File: Calculator/src/CalculatorMain.java [hunk 3]
```diff
@@ -29,13 +30,16 @@ public static void main(String args[])
 			case 2:
 				obj2.sub(num1, num2);
 				break;
-		/*	case 3:
+			case 3:
+				obj3.product(num1, num2);
```
Comment (Salman-003) [inline Calculator/src/CalculatorMain.java:34]: make proper alignment

**Review by Salman-003 — CHANGES_REQUESTED** [review]
Do the correction

File: Calculator/src/CalculatorMain.java (hunk 2 above)
Comment (Pavithran2612) [reply Calculator/src/CalculatorMain.java:18]: corrected

**Pull request merged**